### PR TITLE
Simplify KAFKA install

### PR DESCRIPTION
**Pull request by youngbupark**
# Description

Simplify KAFKA install

## Issue reference

n/a

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] The sample code compiles correctly
* [ ] You've tested new builds of the sample if you changed sample code
* [ ] You've updated the sample's README if necessary

### CODE REVIEW

File: 5.bindings/kafka-non-persistence.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# ------------------------------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------------------------------
+
+# Install 1 replica for e2e test
```
Comment (shalabhms) [inline 5.bindings/kafka-non-persistence.yaml:6]: nit: I think this e2e test can be removed..

File: 5.bindings/kafka-non-persistence.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+replicas: 1
+
+# Disable persistent storage
+persistence:
+  enabled: false
+
+# Topic creation and configuration for dapr test
```
Comment (shalabhms) [inline 5.bindings/kafka-non-persistence.yaml:13]: nit: this "dapr test: ..

Comment (youngbupark) [reply 5.bindings/kafka-non-persistence.yaml]: fixed.

File: 5.bindings/kafka-non-persistence.yaml (hunk 1 above)
Comment (youngbupark) [reply 5.bindings/kafka-non-persistence.yaml]: fixed.

**Pull request merged**